docs/guide/api-environment.md

```diff
@@ -1,16 +1,5 @@
 # Environment API
 
-:::warning Experimental
-Initial work for this API was introduced in Vite 5.1 with the name "Vite Runtime API". This guide describes a revised API, renamed to Environment API. This API will be released in Vite 6 as experimental. You can already test it in the latest `vite@6.0.0-beta.x` version.
-
-Resources:
-
-- [Feedback discussion](https://github.com/vitejs/vite/discussions/16358) where we are gathering feedback about the new APIs.
-- [Environment API PR](https://github.com/vitejs/vite/pull/16471) where the new API were implemented and reviewed.
-
-Please share with us your feedback as you test the proposal.
-:::
-
```

Comment:
The section should not be removed completely. It's still experimental. I think the only change is that it's released.

Reply:
Yeah specifically I think we only need to update this part:

> This API will be released in Vite 6 as experimental. You can already test it in the latest `vite@6.0.0-beta.x` version.

Maybe something like "This API is released in Vite 6 as experimental."

Reply:
Okay, I've finished it